source/initial-dns-seedlist-discovery/tests/uri-with-auth.json

```diff
@@ -0,0 +1,19 @@
+{
+  "uri": "mongodb+srv://auser:apass@test1.test.build.10gen.cc/?replicaSet=repl0",
+  "seeds": [
+    "localhost.test.build.10gen.cc:27017",
+    "localhost.test.build.10gen.cc:27018"
+  ],
+  "hosts": [
+    "localhost:27017",
+    "localhost:27018",
+    "localhost:27019"
+  ],
+  "options": {
+    "auth": {
+      "user": "auser",
+      "password": "apass"
+    }
```

Comment:
I intended for ``options`` to be a flat associative array, and this introduces a nested structure. As these "positional" URI arguments don't really have a name, and hence no "key", I would suggest to add a different top-level item in these tests, for example:

```
"parsed_options": {
    "user": "auser",
    "password": "apassw",
}
```

This then wouldn't break test runners that assume that the items in ``options`` are proper URI options. It might also be needed for drivers to make a mapping, as for example the PHP driver uses ``username`` instead of ``user``.

Reply:
Ah, yeah, that makes sense. I'll update 👍